tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -153,6 +153,17 @@ public function testEventPage(): void {
       'Event body',
       $crawler->filter('#oe-content-body p')->text()
     );
+
+    // Assert inpage_navigation not loaded if there is no body and documents.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = \Drupal::entityTypeManager()->getStorage('node')->load($node->id());
```

Comment:
Do we need to load a fresh instance of `$node`? I don't think it is necessary.
Also, if we really need, we could just 
```suggestion
    $node = Node::load($node->id());
```
and skip the `@var` comment above.